packages/server/modules/activitystream/services/index.js

```diff
@@ -145,13 +153,15 @@ module.exports = {
     return parseInt( res.count )
   },
 
-  async getTimelineCount( { userId, after, before } ) {
-    let query = StreamAcl().count()
+  async getTimelineCount( { userId } ) {
```

Comment:
The `before` and `after` parameters were introduced recently and you probably missed the merging of those :)

Reply:
good catch, how about now ? 😄